src/app/features/shared/components/molecules/Table/Table.tsx

```diff
@@ -52,14 +68,19 @@ const createLoadingData = (numColumns: number, numRows: number = 5) =>
   [...Array(numRows)].map(_ => [...Array(numColumns)].map(_ => ""))
 
 const Table: React.FC<Props> = ({ columns, loading, numLoadingRows, hasFixedColumn, noValuesPlaceholder, className, ...restProps }) => {
-  const data = loading
-    ? createLoadingData(columns.length, numLoadingRows)
-    : restProps.data
+  const data = restProps.data
 
   const fixedColumnWidth = hasFixedColumn
     ? columns[columns.length - 1].minWidth
     : undefined
 
+const onClick = (href: string | undefined, e: any) => { 
+  if (href) {
+    e.stopPropagation()
+    href && navigate(href)
```

Comment:
There is a duplicate check for `href` here. (Inside the if statement)

Reply:
Fixed it